ui/dist/pomerium-horizontal-black.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 28.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 1258.58 187.09" style="enable-background:new 0 0 1258.58 187.09;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#FFFFFF;}
```

Comment:
I think the white background may look bad in dark mode.

Can we get a version of this image that is purple over transparent, like the previous logo image?

Reply:
sure

Reply:
updated with a purple logo